### PR TITLE
Adds Ident Support

### PR DESCRIPTION
**Pull request by IanMitchell**
Currently a bug where it pings someone three times:

```
 ARX-7 Query from CoffeeFlux: add reply #commie-subs +38s
  ARX-7 Admin Query from Jukey: add reply #commie-subs +39s
  ARX-7 Notice: NickServ -> ARX-7: STATUS Jukey 1 +497ms
  ARX-7 Notice: NickServ -> ARX-7: STATUS Jukey 1 +0ms
  ARX-7 Notice: NickServ -> ARX-7: STATUS Jukey 1 +0ms
  ARX-7 Jukey is not identified. +0ms
```

### CODE REVIEW

**Comment by puddi** [comment]
>build process

this is the most effort i've seen anyone put into irc

**Comment by IanMitchell** [comment]
Heheh yeah. ES6 is worth it though! Jukey figured out the bug, working on a fix now: https://twitter.com/kunaldes/status/659165332846739456

File: src/arx7.js [hunk 1]
```diff
@@ -95,108 +95,135 @@ export class ARX7 {
...
+
+          // Search and Verify Plugin Existence
+          let exists = this.commands.some(c => {
+            return c.constructor.name.toLowerCase() === args[1].toLowerCase();
+          });
+
+          if (!exists) {
+            log(`Invalid plugin`);
+            this.client.say(from, `Invalid plugin. ${usage}`);
+            resolves();
```
Comment (puddi) [inline src/arx7.js]: lol

Comment (IanMitchell) [inline src/arx7.js]: Yeah yeah. Code updated :smiley:

**Pull request merged**